httomo/methods_database/packages/external/httomolibgpu/supporting_funcs/prep/phase.py

```diff
@@ -101,7 +101,7 @@ def _calc_memory_bytes_paganin_filter_tomopy(
         + out_slice_size
         + 2 * complex_slice
         + 0.5 * fftplan_slice
-        + 2 * res_slice
+        + res_slice
```

Comment:
The memory hook tests are still failing for the paganin method I think, so why has this been changed?